Purchases/Public/Purchases.swift

```diff
@@ -360,6 +353,9 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
     ) {
 
         Logger.debug(Strings.configure.debug_enabled, fileName: nil)
+        if systemInfo.useStoreKit2IfAvailable {
+            Logger.info(Strings.configure.store_kit_2_enabled, fileName: nil)
```

Comment:
Just added this let me know what you think @aboedo @taquitos

Reply:
Love it